src/models/user.ts

```diff
@@ -49,6 +49,49 @@ export class User {
         return new User(userData.username, userData.password, userData.email, userData.fname, userData.lname);
     }
 
+    /**
+     * Updates users' data, except for the password
+     * @param username username of user
+     * @param email email of user
+     * @param fname first name of user
+     * @param lname last name of user
+     */
+    public static async updateUser(username: string, email: string, fname: string, lname: string) {
+        const data = {
+            username,
+            fname,
+            email,
+            lname,
+        };
+
+        const password = await knex('User').update(data).where('User.username', '=', username).returning('password');
```

Comment:
Why does the password need to be returned?

Reply:
We keep the password attached to the user object in other places, so I thought it'd be best to keep it consistent